src/junit.rs

```diff
@@ -49,12 +60,14 @@ fn get_attribute(e: &BytesStart, name: &str) -> PyResult<Option<String>> {
 
 fn populate(
     rel_attrs: RelevantAttrs,
-    testsuite: String,
+    testsuite: ValidatedString,
     testsuite_time: Option<&str>,
     framework: Option<Framework>,
     network: Option<&HashSet<String>>,
 ) -> PyResult<(Testrun, Option<Framework>)> {
-    let classname = rel_attrs.classname.unwrap_or_default();
+    let classname = rel_attrs
+        .classname
+        .unwrap_or_else(|| ValidatedString::from_string("".to_string()).unwrap());
```

Comment:
Defining a `ValidatedString::new()` to create an empty string would be preferable to this dance with `to_string` and `unwrap`